go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -274,7 +170,7 @@ func Build(statement sqlparser.Statement, tables map[string]*schema.Table) (*Pla
 	case *sqlparser.Set:
 		plan, err = analyzeSet(stmt), nil
 	case *sqlparser.DDL:
-		plan, err = analyzeDDL(stmt, tables), nil
+		plan = &Plan{PlanID: PlanDDL}
```

Comment:
This does not contain the DDL Query. Could you explain the reason for not adding the query?